esp-hal/src/i2s_parallel.rs

```diff
@@ -15,6 +15,10 @@
 //! - 16bit: [A, B, C, D] is output as [B, A, D, C] (i.e., 16bit words are
 //!   swapped)
 //!
+//! I2S0 does not support true 8bit parallel output, so if you want to do 8bit
```

Comment:
Is this every device we have? Otherwise, this note should be `cfg`'d to the device that has this issue.

Reply:
I'll gate this for esp32 (since the esp32s3 only has I2S0). In any case currently this driver is gated for esp32 only at the moment.